### ISSUE TITLE
Inquiry about STICI Model Parameters for Chromosome 22 SV Training.

### ISSUE THREAD

**Issue by tianyongjing-genomics-cn** [shilab/STI#7]
Thank you very much for your team providing us with a great tool.

I am writing to inquire about the specific parameters and reference panel details used in your Nature Communications paper "STICI: Split-Transformer with integrated convolutions for genotype imputation" (https://www.nature.com/articles/s41467-025-56273-3).

I am particularly interested in the experimental setup for training STICI on chromosome 22 structural variants (SVs). 

Specifically, I would like to know:

1. What were the exact hyperparameters used for training STICI on chromosome 22 SVs?

2. If the training dataset contains a limited number of haplotypes, what model parameters should be adjusted to maintain imputation accuracy?

3. When using a phased reference panel to impute unphased samples, the training must be configured for unphased mode. What is the expected reduction in imputation accuracy?

**Comment by ErfanMowlaei** [comment]
@tianyongjing-genomics-cn I have added the colab notebook that we used to run STICI on chromosome 22 SVs (although it is old a bit). You can find it in the examples folder now and check the hyper-parameters.

As for your second question, I would say you should use an embedding dimension of 64 and 128 and see which one would work the best. I hardly doubt other parameters make any big difference or improve the results much.

Lastly, I don't recall running it with unphased data, although I added the capability to do it. If anything, the accuracy shouldn't drop more than 2-5%, but in general I suggest phasing the data first using available prephasing tools.

**Comment by tianyongjing-genomics-cn** [comment]
Thank you for your comprehensive reply. 

I have an additional inquiry about variant representation: in cases where the reference panel contains multiple variants at identical genomic positions (such as SV1 and SV2), is variant ID uniqueness sufficient for training and imputation, or must the genomic coordinates also be distinct?

**Comment by ErfanMowlaei** [comment]
You are most welcome!

Only variant ID is important and the `DataReader` won't look at position and other fields. Just to be safe, please make sure that the variants are sorted based on their positions.
